### DISCUSSION TITLE
Samsung Soundbar Q70T

### DISCUSSION THREAD

**Discussion by robkarpati** [music-assistant/support#618] [Feature requests and ideas]
Please add Samsung Soundbar Q70T to the supported devices

**Comment by marcelveldt** [comment]
Did you use a particular integration (Samsung, DLNA) to add it to HA ?

**Reply by marcelveldt** [reply]
Or is this actually a feature request to support this device ?

**Comment by robkarpati** [comment]
yes, I use DLNA Digital Media Renderer

Marcel van der Veldt ***@***.***> ezt írta (időpont: 2022.
júl. 8., P, 18:52):
